independent-publisher-connectors/Cloverly/apiDefinition.swagger.json

```diff
@@ -0,0 +1,2266 @@
+                "weight": {
+                  "type": "object",
+                  "properties": {
+                    "value": {
+                      "type": "number",
+                      "description": "",
+                      "title": "Value"
+                    },
+                    "units": {
+                      "type": "string",
+                      "description": "",
+                      "title": "Units"
+                    }
+                  },
+                  "title": "Weight",
+                  "required": [
+                    "value",
+                    "units"
+                  ]
+                },
+                "mode": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Mode"
+                },
+                "type": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Type"
+                },
+                "distance": {
+                  "type": "object",
+                  "properties": {
+                    "value": {
+                      "type": "string",
+                      "description": "",
+                      "title": "Value"
+                    },
+                    "units": {
+                      "type": "string",
+                      "description": "",
+                      "title": "Units"
+                    }
+                  },
+                  "title": "Distance"
+                },
+                "project_match": {
+                  "type": "object",
+                  "properties": {
+                    "type": {
+                      "type": "string",
+                      "description": "",
+                      "title": "Type"
+                    },
+                    "location": {
+                      "type": "object",
+                      "properties": {
+                        "postal_code": {
+                          "type": "string",
+                          "description": "",
+                          "title": "Postal Code"
+                        },
+                        "country": {
+                          "type": "string",
+                          "description": "",
+                          "title": "Country"
+                        }
+                      },
+                      "title": "Location"
+                    },
+                    "note": {
+                      "type": "string",
+                      "description": "",
+                      "title": "Note"
+                    }
+                  },
+                  "title": "Project Match"
+                },
+                "note": {
+                  "type": "string",
+                  "description": "",
+                  "title": "Note"
+                }
+              }
+            }
+          }
+        ]
+      }
+    },
+    "/{transaction}/flight": {
+      "post": {
+        "responses": {
+          "default": {

```

Comment:
@troystaylor Default responses are not allowed anymore, which means no dynamic content will be shown in the output and makers have to use parse JSON to parse the response. Please provide valid expected response codes with schema definitions for all the operations in the swagger.